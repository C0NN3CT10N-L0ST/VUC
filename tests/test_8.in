e
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
aliqua.
d
Km rixt py txixt drixpt, gkxj icjmlky rhrloxmpmxci kssptoc spuclxj ixjx km psxgkxf rh rp octtcyc ocijrgkpm.
q