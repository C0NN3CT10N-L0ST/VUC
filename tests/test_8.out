 ____________________________ 
|  __      ___    _  _____   |
|  \ \    / / |  | |/ ____|  |
|   \ \  / /| |  | | |       |
|    \ \/ / | |  | | |       |
|     \  /  | |__| | |____   |
|      \/    \____/ \_____|  |
|                            |
|       Encoder/Decoder      |
|  ------------------------  |
|         |Options|          |
|         e - encode         |
|         d - decode         |
|          q - quit          |
|____________________________|
Choose option:
> 
Enter text to encode (MAX length is 300 characters):

Encoded text
-----------------------------
Sclrt xfjkt ycscl jxm ptrm, ocijromrmkl pyxfxjoxiz rsxm, jry yc rxkjtcy mrtfcl xioxyxykim km spuclr rm ycsclr tpzip psxgkp.
-----------------------------
 ____________________________ 
|  __      ___    _  _____   |
|  \ \    / / |  | |/ ____|  |
|   \ \  / /| |  | | |       |
|    \ \/ / | |  | | |       |
|     \  /  | |__| | |____   |
|      \/    \____/ \_____|  |
|                            |
|       Encoder/Decoder      |
|  ------------------------  |
|         |Options|          |
|         e - encode         |
|         d - decode         |
|          q - quit          |
|____________________________|
Choose option:
> 
Enter text to decode (MAX length is 300 characters):

Decoded text
-----------------------------
Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-----------------------------
 ____________________________ 
|  __      ___    _  _____   |
|  \ \    / / |  | |/ ____|  |
|   \ \  / /| |  | | |       |
|    \ \/ / | |  | | |       |
|     \  /  | |__| | |____   |
|      \/    \____/ \_____|  |
|                            |
|       Encoder/Decoder      |
|  ------------------------  |
|         |Options|          |
|         e - encode         |
|         d - decode         |
|          q - quit          |
|____________________________|
Choose option:
> 
Exiting program...